docs/spec.yml

```diff
@@ -254,7 +367,100 @@ components:
         - scope
         - root_token_enabled
         - csrf_enabled
-  
+    Job:
+      type: object
+      properties:
+        id:
+          type: string
```

Comment:
I leave type string for id field to be more flexible for possible changes in job id generation in the future, if it will be needed